testgrid/config/config.yaml

```diff
@@ -1456,6 +1458,8 @@ dashboards:
     test_group_name: kubernetes-e2e-gke-updown
   - name: gci-gke-updown
     test_group_name: kubernetes-e2e-gci-gke-updown
+  - name: gke-gci-ci-master
```

Comment:
@ixdy I'd be fine with putting it on the `google-gci-dev` gridboard. In that case we can probably change the tab name to something like `ci-gke-master`, since everything on that gridboard is `gci`.